_pages/learn.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: splash
+permalink: /learn.html
+title: "Python Tutorials & Learning Resources for Scientists"
+classes: flowing
+header:
+  overlay_image: images/header.png
+  overlay_filter: 0.6
+scientists:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Our catalog of vetted open source tools makes it easier for scientists to find the trusted tools that they need to develop their open science workflows."
+    title: "Help scientists find the open science tools that they need"
+    url: https://www.pyopensci.org/python-packages.html
+    btn_label: View our growing list of accepted scientific Python packages
+    btn_class: btn--primary
+overview:
+  - excerpt: "pyOpenSci creates resources to help you navigate the Python packaging ecosystem with ease. All of our materials are community developed and go through extensive review."
+development:
+   - image_path: images/pyopensci-learn-community-created.png
+     alt: "alt here "
+     title: "Community Developed Python Tutorials for Scientists"
+     excerpt: "Our tutorials are created through a multi-stage community review process.
+
+     * Tutorials are developed by the pyOpenSci team or community members.
+
+     * Tutorials are reviewed by tool maintainers to ensure ideas and concepts are accurate
+
+     * Before publication, tutorials then go through several rounds of community review for accuracy, usability and accessibility."
+whats_next:
+  - image_path: images/pyopensci-learn-github-collaboration.png
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Through the Better software for science fellowship we will be developing lessons focused on teaching core Github collaboration skills needed for both contributing to open source software and for working in an open science team environment. "
+    title: "What's next: Collaborative GitHub for Scientists"
+toc: false
+---
+
+{% include feature_row id="overview" type="center" %}
+
+<div class="pyos-section purple">
+<div class="content" markdown="1">
+
+{% include feature_row id="development" type="left" %}
+
+</div>
+</div>
+
+{% include div_purple_bottom.html  %}
+
+<div class="pyos-section">
+<div class="content" markdown="1">
+
+## Beginner friendly Python packaging tutorials
```

Comment:
```suggestion
## Beginner-friendly Python packaging tutorials
```

Reply:
one thing that I think is confusing about the layout is that it's not clear that these tutorials are sequential, nor do I know which one to start with. the way they're laid out on the page, they look like I can pick and choose which one to start with, but once I click-through it's clear that they're part of a sequence.

I think what I'm struggling with is that in my mind, I think of a tutorial as being the entire sum of these lessons, which teach how to create a python package. one way to address this would be to add more copy that adds clarification around what someone can expect when they click through, or making it clear that these are lessons from a larger tutorial. alternatively (or in addition to), numbering each of the lessons would help indicate that these are part of a bigger tutorial, and can//should be done sequentially.

Reply:
the "add metadata using a pyproject.toml file...." title shows up twice, at the beginning and end of the list, but clearly links to two separate lessons. it looks like the last lesson should be titled "Make your Python package PyPI ready" or "updated metadata in pyproject.toml"

Reply:
I'm not sure how much I like the term "beginner-friendly". I would lean more toward "guided" or "hands-on".